lib/widgets/content.dart

```diff
@@ -36,7 +105,9 @@ class MessageContent extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return InheritedMessage(message: message,
-      child: BlockContentList(nodes: content.nodes));
+      child: DefaultTextStyle(
```

Comment:
The content elements that had been getting these attributes from
    TextTheme.bodyMedium (
    
      fontSize: 14.0
      height: 1.43
    
    ), and are now getting them from here (
    
      fontSize: 17
      height: 22 / 17 which is ~1.29
    
    ), are:
    
    - line breaks (for `LineBreakNode`),
    - code blocks (and our simplified math blocks), and
    - list-item markers.
    
    So, those all look different after this change.

In your screenshots, I see the difference in the list-item markers. I don't see a difference in the code blocks; did their font size end up changing, or was that change suppressed by something else?

Reply:
Oh, oops: yeah, code blocks set their own font size (0.825 * `kBaseFontSize`), so that doesn't change in this commit. Code blocks don't set their line height, so that does change, from 1.43x to ~1.29x. But then a subsequent commit sets it to 1.4x, which is very close to the original 1.43x.